docs/reference/release-notes/7.2.asciidoc

```diff
@@ -220,6 +220,14 @@ Machine Learning::
 * Add task recovery on node change  {pull}39416[#39416]
 * Stop tasks on failure {pull}39203[#39203]
 * Add _preview endpoint {pull}38924[#38924]
+* Remove hard limit for maximum forecast interval and limit based on the time interval of data added
```

Comment:
This change had both a Java and a C++ component.  The Java change is release noted as "Increase maximum forecast interval to 10 years" 8 lines above.

What do you think about combining them @lcawl.  Maybe delete the two current items and replace with something like:

>* Increase maximum forecast interval from 8 weeks to a limit based on the amount of data seen, up to a maximum of 10 years {ml-pull}214[#214] and {pull}41082[#41082] (issue: {issue}41103[#41103])

But I'll leave the exact wording up to you.

Reply:
Thanks @droberts195 I committed that change.